global.json

```diff
@@ -29,10 +29,10 @@
       "allowPrerelease": false
     },
```

Comment:
Delete L27-30 as the actual required VS version now is 17.8 which is already provided via the build tool machines.